model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -66,6 +71,21 @@ def allocate_data(backend, input_data):
     return input_data
 
 
+def _match_marker(marker, backend):
+    for m in marker:
+        match m.markname:
+            case "embedded_remap_error" if is_embedded(backend):
+                pytest.xfail("Embedded backend currently fails in remap function.")
+            case "embedded_as_offset_error" if is_embedded(backend):
```

Comment:
```suggestion
            case "uses_as_offset" if is_embedded(backend):
```
would be more in the spirit of the gt4py markers